frappe/utils/oauth.py

```diff
@@ -166,22 +164,12 @@ def get_info_via_oauth(provider, code, decoder=None, id_token=False):
 
 
 def login_oauth_user(
-	data=None, provider=None, state=None, email_id=None, key=None, generate_login_token=False
+	data: dict | str,
+	*,
+	provider: str | None = None,
+	state: dict | str,
+	generate_login_token: bool = False,
```

Comment:
Well, it does look like an internal function. If anyone is using it, best to break it this way rather than adding other maintenance overheads for their sake 😄